common/pulp/common/constants.py

```diff
@@ -48,7 +49,19 @@
 PRIMARY_ID = '___/primary/___'
 
 # this is used by both platform and plugins to find the default CA path
-DEFAULT_CA_PATH = '/etc/pki/tls/certs/ca-bundle.crt'
+DEFAULT_CA_PATH_LIST = [
+    '/etc/pki/tls/certs/ca-bundle.crt',    # Fedora/RHEL
+    '/etc/ssl/certs/ca-certificates.crt',  # Debian/Ubuntu/Gentoo etc.
+    '/etc/ssl/ca-bundle.pem',              # OpenSUSE
+    '/etc/pki/tls/cacert.pem',             # OpenELEC
+]
+# set DEFAULT_CA_PATH based on DEFAULT_CA_PATH_LIST for backward compatibility with plugins
+for path in DEFAULT_CA_PATH_LIST:
+    if os.path.exists(path):
+        DEFAULT_CA_PATH = path
+        break
+else:
+    DEFAULT_CA_PATH = None
```

Comment:
It just occurred to me that it might be nice to have a comment on this, indicating that no CA path was found, and we probably need a new entry in the CA path list. What do you think?